storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -1081,6 +1088,14 @@ public Nimbus(Map<String, Object> conf, INimbus inimbus, IStormClusterState stor
             BlobStore blobStore, TopoCache topoCache, ILeaderElector leaderElector, IGroupMappingServiceProvider groupMapper)
         throws Exception {
         this.conf = conf;
+
+        this.metricsStore = null;
+        try {
+            this.metricsStore = MetricStoreConfig.configure(conf);
+        } catch (Exception e) {
+            LOG.error("Failed to initialize metric store", e);
```

Comment:
Could we add in a comment that the metrics store is not critical to the operation of the cluster so if it does not come up we will not stop the cluster from coming up.